Sources/DNA/Color/Color.swift

```diff
@@ -189,29 +86,16 @@ import UIKit
         return UIColor(hex: "#262633")
     }
 
-    // swiftlint:disable:next identifier_name
-    convenience init?(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) {
-        self.init(red: r / 255.0, green: g / 255.0, blue: b / 255.0, alpha: a)
+    public class var accentSecondaryBlue: UIColor {
```

Comment:
Why do we need these `accent` colors here?

Reply:
Good question, I don't know why do we use the accent colors, I just move them here because they have no usage in Bootstrap.